goal_src/jak1/engine/gfx/font.gc

```diff
@@ -1830,3 +1830,49 @@
     (draw-string str buf context)
     )
   )
+
+(defun draw-string-xy-scaled ((str string) (buf dma-buffer) (x int) (y int) (color font-color) (flags font-flags) (scale float))
```

Comment:
Write something that says this was added for the pc port.

I think the way you wrote this function is effectively a wrapper for `print-game-text-scaled`. I think you can just copy `draw-string-xy` but simply set a scale on the `font-context` before drawing instead of copying all of this unnecessary text padding stuff.

Reply:
Will add a note about modified for PC 👍

So `print-game-text-scaled` calls `print-game-text` which ultimately calls `draw-string`, so this actually feels like a bit more of a direct route to a scaled call to `draw-string`.

I'll try removing the extra padding code and see how things look today 👍

Reply:
OK yep as you said, the adjustments/padding were unnecessary - looked exactly the same with and without (I did actually test with `(font-flags left)` and `(font-flags middle)`